components/AirdropForm.tsx

```diff
@@ -48,7 +47,7 @@ function formatWaitTime(seconds: number): string {
 
 export const AirdropForm = ({ className }: AirdropFormProps) => {
   const toaster = useToast();
-  const amountOptions = useMemo(() => [0.05, 0.1, 0.15, 0.25], []);
+  const amountOptions = useMemo(() => [200, 500, 1000, 2000], []);
```

Comment:
Can we set this as a constant somewhere so that we only need to update one place for changes?